examples/contrib/sops/workflow-with-age/README.md

```diff
@@ -0,0 +1,84 @@
+# sops: AGE Workflow example
+
+### Overview
+
+This example demonstrates invocation of `sops` KRM config function to decrypt and encrypt all other resources using the
+already existing AGE keys. That includes decryption and encryption of some meta-resources, e.g.
+[apply-setters.yaml](apply-setters.yaml), that is a setter configuration used for rendering and that may contain
+sensitive information, e.g. passwords, keys & etc and that may be necessary to keep encrypted in git repo.
+
+The `sops` KRM config function encrypts and decrypts resources using the sops tool. Learn more on the [sops website].
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/contrib/sops/workflow-with-age
+```
+
+The package resources, e.g. [deployment.yaml](deployment.yaml) as well as some meta-resources, e.g. [apply-setters.yaml](apply-setters.yaml)
+are stored in encrypted form in git repo.
+
+### Function invocation
+
+Invoke the decryption by running the following command:
+
+```shell
+$ kpt fn eval \
+        --fn-config workflow-with-age/decrypt.yaml \
+        --env SOPS_IMPORT_AGE="$(cat workflow-with-age/age_keys.txt)" \
+        --image-pull-policy ifNotPresent \
```

Comment:
Please drop this line in the user-facing doc.
We can leave it to use the default value for this flag.